examples/builder_derive_interop/1_augment_args.md

```diff
@@ -0,0 +1,7 @@
+*[Jump to source](1_augment_args.rs)*
+
+# 1: Augment built `Command` with derived `Args`
```

Comment:
I'd recommend putting the titles into user-focused language.  Maybe something like "Using Derived Arguments in Your Builder application"